src/core.js

```diff
@@ -1327,6 +1327,10 @@ define([
             var foo = _this.getInsertTargetAndPosition(
                 _this.getCurrentlySelectedMug(), qType);
             mug = _this.data.core.form.createQuestion(foo[0], foo[1], qType);
+            var $firstInput = _this.$f.find(".fd-question-properties input:text:visible:first");
+            if ($firstInput.length) {
+                util.setCaretPosition($firstInput[0], 0, $firstInput.val().length);
```

Comment:
This line can be shortened to `$firstInput.focus().select();`, a more standard way to do this.